Automattic/wp-calypso
# Playwright script to take screenshot of the login page

#### Changes proposed in this Pull Request

For context, check pau2Xa-3my-p2.

* Playwright script that takes a full page screenshot of login, signup, and checkout pages. 
* The screenshots are uploaded to a custom WP.com REST API, which in turn uploads them to a WP.com blog. 

#### Testing instructions



Depends on D69547-code.

* Follow the instructions in p4TIVU-8Qw-p2 to setup e2e tests locally
* Checkout this branch in your local Calypso instance
* In test/e2e folder of your local Calypso folder, run `yarn install`
* Apply D69547-code and follow the setup instructions written in that patch. 
* Run the Playwright script using `BABEL_ENV=test yarn jest specs/specs-playwright/wp-signup__screenshot.ts`.
* Verify that fullpage screenshots of signup, login, and checkout are stored on the blog, and the ToS link is visible in all these pages.



Related to #
